paper-server/src/main/java/org/bukkit/craftbukkit/help/CustomIndexHelpTopic.java

```diff
@@ -7,6 +7,7 @@
 import org.bukkit.help.HelpMap;
 import org.bukkit.help.HelpTopic;
 import org.bukkit.help.IndexHelpTopic;
+import org.jetbrains.annotations.NotNull;
```

Comment:
```suggestion
import org.jspecify.annotations.NonNull;
```
You should use the jspecify annotations in this case

Reply:
Kept jetbrains one since the parent method also has a jetbrains one and I didn't feel like it was a needed change for this PR. Can update this one and parent's ones if needed though.

Reply:
Then i suppose this can be kept as-is, paper will replace them later when they merge the checkstyle pr